test/releasers/rust.ts

```diff
@@ -0,0 +1,251 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as assert from 'assert';
+import {describe, it, afterEach} from 'mocha';
+import * as nock from 'nock';
+import {Rust} from '../../src/releasers/rust';
+import * as snapshot from 'snap-shot-it';
+import * as suggester from 'code-suggester';
+import * as sinon from 'sinon';
+import {readPOJO, stringifyExpectedChanges} from '../helpers';
+import {readFileSync} from 'fs';
+import {resolve} from 'path';
+
+nock.disableNetConnect();
+const sandbox = sinon.createSandbox();
+const fixturesPath = './test/releasers/fixtures/rust';
+
+describe('Rust', () => {
+  afterEach(() => {
+    sandbox.restore();
+  });
+  describe('run', () => {
+    it('creates a release PR for non-monorepo', async () => {
+      const releasePR = new Rust({
+        repoUrl: 'fasterthanlime/rust-test-repo',
+        releaseType: 'rust',
+        packageName: 'crate1',
+        apiUrl: 'https://api.github.com',
+      });
+
+      // Indicates that there are no PRs currently waiting to be released:
+      sandbox
```

Comment:
thanks for putting in this work, I'm finding this pattern is more readable.

Reply:
Agreed, I feel a lot better about those tests now!